tensorflow/compiler/mlir/lite/transforms/prepare_tf.cc

```diff
@@ -81,13 +81,52 @@ class PrepareTFPass : public PassWrapper<PrepareTFPass, FunctionPass> {
   bool unfold_batch_matmul_;
 };
 
+
+template <class TFFakeQuantOp>
+struct FetchConstantMinMaxInputs {
+
+  using AttrType = DenseFPElementsAttr;
+  bool operator () (TFFakeQuantOp tf_op, AttrType &min_value, AttrType &max_value) const {
+    Value min = tf_op.min(), max = tf_op.max();
+    ;
+    // TODO This is likely redundant (Identity elimination rule are in
```

Comment:
not really. The identity elimination runs after the pattern, because the pattern matching work queue works backwards.

Reply:
Done.  I've left the TODO comment w.r.t. non-coverage of chained Identity and IdentityN operators.  Let me know if you prefer that removed also.